File: src/main/java/org/goafabric/invoice/process/adapter/patient/dto/MedicalRecordType.java
package org.goafabric.invoice.process.adapter.patient.dto;

public enum MedicalRecordType {

    ANAMNESIS("ANAMNESIS"),
    CONDITION("CONDITION"),
    CHARGEITEM("CHARGE"),
    FINDING("FINDING"),
    THERAPY("THERAPY"),
    BODY_METRICS("BODY_METRICS");
    
    private String value;

    MedicalRecordType(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }



}

File: src/main/kotlin/org/goafabric/invoice/process/adapter/patient/dto/MedicalRecordType.kt
package org.goafabric.invoice.process.adapter.patient.dto

enum class MedicalRecordType(val value: String) {
    ANAMNESIS("ANAMNESIS"),
    CONDITION("CONDITION"),
    CHARGEITEM("CHARGE"),
    FINDING("FINDING"),
    THERAPY("THERAPY"),
    BODY_METRICS("BODY_METRICS")
}
